pkg/workloadmeta/collectors/internal/process/process.go

```diff
@@ -0,0 +1,141 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package process
+
+import (
+	"context"
+	"time"
+
+	"github.com/benbjohnson/clock"
+
+	"github.com/DataDog/datadog-agent/pkg/config"
+	dderrors "github.com/DataDog/datadog-agent/pkg/errors"
+	"github.com/DataDog/datadog-agent/pkg/process/checks"
+	workloadmetaExtractor "github.com/DataDog/datadog-agent/pkg/process/metadata/workloadmeta"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+	"github.com/DataDog/datadog-agent/pkg/workloadmeta"
+)
+
+const collectorId = "process-agent"
+
+const collectionInterval = 1 * time.Minute
```

Comment:
Not necessarily something for this PR, but should this be configurable e.g. to increase the collection interval if we find it's having a performance impact?

Reply:
I think so. It's also a small change so I can add it in this PR.